src/api.ts

```diff
@@ -32,15 +33,16 @@ export const bas:BasToolkit = {
         });
     },
 
-    getAction: (actionId: string) => ActionsController.getAction(actionId),
-    getParameter,
-    performAction,
+    //getAction: (actionId: string) => ActionsController.getAction(actionId),
+    //getParameter,
+    //performAction,
+    isLCAPEnabled,
 
```

Comment:
Why do you mark these functions as a comment? they are used in many places, including other repositories

Reply:
as far as i understand from rotem i need to remove all the action and getparameters calls
@idoprz can you also got over the code